src/java/org/apache/cassandra/repair/RepairJob.java

```diff
@@ -97,7 +101,7 @@ public RepairJob(RepairSession session, String columnFamily)
 
     public long getNowInSeconds()
     {
-        long nowInSeconds = FBUtilities.nowInSeconds();
+        long nowInSeconds = ctx.clock().currentTimeMillis() / 1000l;
```

Comment:
nit: Clock has a `nowInSeconds()` method already, right?

Also, my OCD says change `Clock#nowInSeconds()` to use `1000L` instead of `1000l` ;)